.travis.yml

```diff
@@ -4,9 +4,12 @@ env:
     - DATABASE_URL='postgres://postgres@localhost/user_management'
     - TERM='xterm-256color'
   matrix:
-    - TOX_ENV=py36-django111
-    - TOX_ENV=py37-django111
-    - TOX_ENV=py38-django111
+    - TOX_ENV=py36-django22
```

Comment:
It's easier to upgrade when libraries support two versions of Django side-by-side, so if it's possible to avoid dropping Django 1.11 here (just for one release), it should make things easier for some.